FluentUI.Demo/src/androidTest/java/com/microsoft/fluentuidemo/demos/V2PeoplePickerUITest.kt

```diff
@@ -107,9 +110,11 @@ class V2PeoplePickerUITest {
             )
         }
 
-        val peoplePickerChip = composeTestRule.onNodeWithText("firstName", true, useUnmergedTree = true)
+        val peoplePickerChip =
+            composeTestRule.onNodeWithText("firstName", true, useUnmergedTree = true)
         peoplePickerChip.performClick()
-        val cancelButton = composeTestRule.onNodeWithContentDescription("Close", useUnmergedTree = true)
+        val cancelButton =
```

Comment:
Space formatting

Reply:
Formatting file change. not added intentionally.